market/connector/twitter/README.md

```diff
@@ -8,6 +8,7 @@ See what’s happening in the world right now
 
 1. Shows how to read recent tweets with a keyword.
 ![read-tweets](doc/img/demo_readTweets.png)
```

Comment:
something here with the inline is strange -> readme view

Reply:
fixed